modules/axhal/src/arch/riscv/cache.rs

```diff
@@ -0,0 +1,13 @@
+#![allow(unused)]
+
+use core::ptr::NonNull;
+
+/// Invalidate data cache
+pub fn dcache_invalidate_range(_addr: NonNull<u8>, _size: usize) {
+    //TODO
```

Comment:
unimplemented!

Reply:
It can work with qemu, because virt machine cache is synced, but will not work on real machine. I am not familiar with riscv, so need someone else to impl this.

Reply:
> It can work with qemu, because virt machine cache is synced, but will not work on real machine. I am not familiar with riscv, so need someone else to impl this.

yep, I mean you should use `unimplemented()` macros explicitly, in case if someone want to use this API but failed to get the expected result.

Reply:
If use `unimplemented!()`, x86 qemu will not work, but it can work.